### ISSUE TITLE
Patch did not work for me on High Sierra

### ISSUE THREAD

**Issue by gigantaur** [thebitguru/play-button-itunes-patch#44]
Hi Farhan,

The patch has worked for me for many years however it i no longer working on High Sierra.

I notice it stopped working, downloaded 1.1.  Disabled SIP. Rebooted.  Ran 1.1.  It said patched already - I rewatched for the hell of it.  Rebooted. iTunes still launches using the play button.  

Any help would be greatly appreciated.

Cheers,
Chris

See screenshot
![screen shot 2018-04-18 at 10 52 43 pm](https://user-images.githubusercontent.com/6449584/38933045-3d35afbe-435b-11e8-95ed-93ec4b9fc56c.png)

**Comment by gigantaur** [comment]
Sorry here are the logs ..

[com.thebitguru.Play-Button-iTunes-Patch 2016-11-09 12-00.log](https://github.com/thebitguru/play-button-itunes-patch/files/1924068/com.thebitguru.Play-Button-iTunes-Patch.2016-11-09.12-00.log)
[com.thebitguru.Play-Button-iTunes-Patch 2018-04-18 12-38.log](https://github.com/thebitguru/play-button-itunes-patch/files/1924069/com.thebitguru.Play-Button-iTunes-Patch.2018-04-18.12-38.log)

**Comment by gigantaur** [comment]
Disabling the Play button (and other useful buttons) in Preferences/Keyboard/"Use F1 F2 etc as standard function keys"  did work ..

**Comment by thebitguru** [comment]
Hi, @gigantaur, sorry, I have been quite busy lately.  There is a known issue with High Sierra that I need to work on and this is likely because of it.  I will try to get to this soon.

**Comment by gigantaur** [comment]
Thanks heaps for the reply.

A fantastic contribution that you make in doing this - one many of us really appreciate.

Do you think you will be able to avoid the need recovery mode to turn off the system protection - even a sudo command line script to start or finish would be better than the whole recovery mode ..

Cheers
Chris


On 26 Apr 2018, at 4:36 am, Farhan Ahmad <notifications@github.com> wrote:

Hi, @gigantaur, sorry, I have been quite busy lately. There is a known issue with High Sierra that I need to work on and this is likely because of it. I will try to get to this soon.

**Comment by rkcarter** [comment]
@gigantaur, I don’t think that will work. One of the points of System Integrity Protection is that a remote attacker can’t modify the system code, even if they acquire root.

- Rick

--
Rick Carter
Ann Arbor, Michigan USA
Rick.Carter@gmail.com

**Comment by aaether32323** [comment]
Until the patch is updated one can always run
`sudo chmod -x /Applications/iTunes.app/Contents/MacOS/iTunes` to completely turn off iTunes. Of course this doesn't work if you actually use iTunes.

**Comment by aaether32323** [comment]
@AlbertoFdzM I forgot that you need to disable SIP for that to work
